obs/preinstallimage-bios.sh.in

```diff
@@ -725,6 +725,15 @@ BindsTo=fty-license-accepted.service
 After=fty-license-accepted.service
 EOF
 
+# Remove nut.target from multi-user.target, to avoid breaking
```

Comment:
Maybe move this comment inside the unit file? its name is pointless, content near non existent, purpose will be questionable after we forget ;)